dvc/command/ls/__init__.py

```diff
@@ -37,10 +37,10 @@ def run(self):
             if self.args.show_json:
                 import json
 
-                logger.info(json.dumps(entries))
+                ui.write(json.dumps(entries))
             elif entries:
-                entries = _prettify(entries, sys.stdout.isatty())
-                logger.info("\n".join(entries))
+                entries = _prettify(entries, with_color=True)
```

Comment:
We should always be printing in colors and we should let `colorama` to handle appropriate usage. It might help us with `--color-always` and `--no-color` flags (if we introduced it).

Reply:
I have been thinking of moving to rich more and more to not deal with colors, pagers, etc. But,it's a bit slow, especially for `ls`. For 27000 files, `ui.write` is 45X  faster (55ms vs 2.5s).